src/data/finset/lattice.lean

```diff
@@ -673,9 +673,16 @@ lemma supr_finset_image {f : γ → α} {g : α → β} {s : finset γ} :
   (⨆ x ∈ s.image f, g x) = (⨆ y ∈ s, g (f y)) :=
 by rw [← supr_coe, coe_image, supr_image, supr_coe]
 
-lemma sup_finset_image {f : γ → α} {g : α → β} {s : finset γ} :
+lemma sup_finset_image {β : Type*} {γ : Type*}
+  [semilattice_sup_bot β] (f : γ → α) (g : α → β) (s : finset γ) :
   s.sup (g ∘ f) = (s.image f).sup g :=
```

Comment:
I know this is out of scope for this change, but this lemma feels backwards to me given its name and the lemma statement above.